homeassistant/components/qwikswitch.py

```diff
@@ -28,6 +29,32 @@
 CONF_BUTTON_EVENTS = 'button_events'
 CV_DIM_VALUE = vol.All(vol.Coerce(float), vol.Range(min=1, max=3))
 
+
+def _validate_sensor(config):
+    """Validate sensor."""
+    try:
+        from pyqwikswitch import SENSORS
```

Comment:
I don't think this is a good idea. Since requirements are installed after config validation, any upgrade to the library and sensor schema will not work since the new config will be validated by the old schema.

Reply:
During upgrades deps are removed, the idea was that it just does not validate first time during upgrade,but at least during other changes/restarts

Let me rather move this to `setup`, since it is not critical

Reply:
Side note: Deps aren't removed during upgrades.